### PR TITLE
US-9 As a prey I want to see distance to the predators

### PR DESCRIPTION
**Pull request by ProtectedVariable**

### CODE REVIEW

**Review by corentin-junod — APPROVED** [review]
Nice ! En regardant comme ca j'ai rien à dire. C'est super cool d'avoir fait un peu de clean de manière générale !

File: app/src/main/java/ch/epfl/sdp/game/PreyActivity.kt [hunk 1]
```diff
@@ -1,132 +1,134 @@
...
     }
 
-    companion object {
-        private const val LOCATION_REFRESH_DISTANCE = 5
-        private const val LOCATION_REFRESH_TIME = 1000
-        private const val MY_PERMISSIONS_REQUEST_ACCESS_FINE_LOCATION = 10
+    override fun onProviderEnabled(provider: String) {
     }
+
+    override fun onProviderDisabled(provider: String) {
```
Comment (HVYVince) [inline app/src/main/java/ch/epfl/sdp/game/PreyActivity.kt:135]: Are empty functions necessary ? Can't we change the interface if it contains unused functions ?

**Review by HVYVince — APPROVED** [review]
We'll have to refactor later to test if the clicks in test actually do something, not only perform them

File: app/src/main/java/ch/epfl/sdp/game/PreyActivity.kt (hunk 1 above)
Comment (ProtectedVariable) [reply app/src/main/java/ch/epfl/sdp/game/PreyActivity.kt:135]: They may be needed in some other implementation so i'd let them

**Pull request merged**